pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -67,7 +66,9 @@ let
 
     Use pkgs.tree-sitter-grammars.<name> to access.
   */
-  builtGrammars = lib.mapAttrs (_: lib.makeOverridable buildGrammar) grammars;
+  builtGrammars = lib.mapAttrs (_: lib.makeOverridable buildGrammar) (
+    lib.filterAttrs (_: g: !(g.meta.broken or false)) grammars
```

Comment:
This is the wrong location to fix this, as changing this would cause us having a diffrent package list depending on platforms.

Instead it shouod be changed at the location where this is getting used:

https://github.com/NixOS/nixpkgs/blob/1306659b587dc277866c7b69eb97e5f07864d8c4/pkgs/applications/editors/emacs/elisp-packages/manual-packages/treesit-grammars/package.nix#L23